bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -285,7 +280,22 @@ class AudioManager {
       new window.AudioContext() :
       new window.webkitAudioContext();
 
-    return this.listenOnlyAudioContext.createMediaStreamDestination().stream;
+    // Create a placeholder buffer to upstart audio context
+    const pBuffer = this.listenOnlyAudioContext.createBuffer(2, this.listenOnlyAudioContext.sampleRate * 3, this.listenOnlyAudioContext.sampleRate);
+
+    var dest = this.listenOnlyAudioContext.createMediaStreamDestination();
+
+    let audio = document.querySelector(MEDIA_TAG);
+
+    // Play bogus silent audio to try to circumvent autoplay policy on Safari
+    audio.src = 'resources/sounds/silence.mp3'
+
+    audio.play().then(() => {
+      return dest.stream;
+    }).catch(e => {
+      logger.warn('Error on playing test audio:', e);
+      return dest.stream;
```

Comment:
Not sure if we can return this if error

Reply:
He is returning the outer promise `audio.play` returning something on the `then/catch` clause does nothing here :)